Documentation/network/bgp-control-plane/bgp-control-plane-v2.rst

```diff
@@ -0,0 +1,765 @@
+continues to forward traffic during Agent restart, so there is no traffic
+disruption.
+
+Optionally, you can use the ``restartTimeSeconds`` parameter. ``RestartTime`` is the time
+advertised to the peer within which Cilium BGP control plane is expected to re-establish
+the BGP session after a restart. On expiration of ``RestartTime``, the peer removes
+the routes previously advertised by the Cilium BGP control plane.
+
+.. code-block:: yaml
+
+    apiVersion: cilium.io/v2alpha1
+    kind: CiliumBGPPeerConfig
+    metadata:
+      name: cilium-peer
+    spec:
+      gracefulRestart:
+        enabled: true
+        restartTimeSeconds: 15
+
+When the Cilium Agent restarts, it closes the BGP TCP socket, causing the emission of a
+TCP FIN packet. On receiving this TCP FIN, the peer changes its BGP state to ``Idle`` and
+starts its ``RestartTime`` timer.
+
+The Cilium agent boot up time varies depending on the deployment. If using ``RestartTime``,
+you should set it to a duration greater than the time taken by the Cilium Agent to boot up.
+
+Default value of ``RestartTime`` is 120 seconds. More details on graceful restart and
+``RestartTime`` can be found in `RFC-4724`_ and `RFC-8538`_.
+
+.. _RFC-4724 : https://www.rfc-editor.org/rfc/rfc4724.html
+.. _RFC-8538 : https://www.rfc-editor.org/rfc/rfc8538.html
+
+.. _bgp_peer_configuration_afi:
+
+Address Families
+----------------
+
+The ``families`` field is a list of AFI (Address Family Identifier), SAFI (Subsequent Address
+Family Identifier) pairs, and advertisement selector. The only AFI/SAFI options currently supported are
+``{afi: ipv4, safi: unicast}`` and ``{afi: ipv6, safi: unicast}``.
+
+By default, if no address families are specified, BGP Control Plane sends both IPv4 Unicast and IPv6 Unicast
+Multiprotocol Extensions Capability (`RFC-4760`_) to the peer.
+
+In each address family, we can control the route publication via ``advertisements`` label selector.
+Various advertisements types are defined :ref:`here <bgp-adverts>`.
+
+.. note::
+
+    Without matching advertisements, no prefix will be advertised to the peer.
+    Default configuration is to not advertise any prefix.
+
+.. _RFC-4760 : https://www.rfc-editor.org/rfc/rfc4760.html
+
+.. code-block:: yaml
+
+    apiVersion: cilium.io/v2alpha1
+    kind: CiliumBGPPeerConfig
+    metadata:
+      name: cilium-peer
+    spec:
+      families:
+        - afi: ipv4
+          safi: unicast
+          advertisements:
+            matchLabels:
+              advertise: "bgp"
+        - afi: ipv6
+          safi: unicast
+          advertisements:
+            matchLabels:
+              advertise: "bgp"
+
+
+.. _bgp-adverts:
+
+BGP Advertisements
+==================
+
+``CiliumBGPAdvertisement`` resource is used to define various advertisement types and attributes
+associated with them. Address family ``advertisements`` label selector in :ref:`peer configuration
+<bgp_peer_configuration_afi>` may match one or more of advertisement resources.
```

Comment:
```suggestion
The ``CiliumBGPAdvertisement`` resource is used to define various advertisement types and attributes
associated with these types. Address family ``advertisements`` label selector in :ref:`peer configuration
<bgp_peer_configuration_afi>` may match one or more advertisement resources.
```

What does “Address family ``advertisements`` label selector” mean?
I find it a bit unclear, could you rephrase or detail?